collectors/scm/gitlab/src/main/java/com/capitalone/dashboard/gitlab/GitlabUrlUtility.java

```diff
@@ -52,7 +66,7 @@ public URI buildApiUrl(GitlabGitRepo repo, boolean firstRun, int resultsPerPage)
         String protocol = getProtocol();
 		String repoName = getRepoName(repoUrl);
 		String host = getRepoHost();
-		String date = getDateForCommits(repo, firstRun);
+		String date = getSinceDate(repo, firstRun);
```

Comment:
The naming here doesn't make sense to me. Reading `getSinceDate` doesn't give me a lot of context of what the method is actually doing. Especially since the method itself is harder to read quickly.

Reply:
Is this one ok?

https://github.com/Hygieia/Hygieia/pull/2755/commits/3cbb00eff8b1daf70388ea69d7c1d03eb0682045#diff-a1c5a6b5d72aaed0621398f4d715f22bR305